zrml/global-disputes/src/benchmarks.rs

```diff
@@ -281,7 +284,16 @@ benchmarks! {
             }
             .into(),
         );
-        assert!(T::Currency::free_balance(&reward_account) == 0u128.saturated_into());
+        #[cfg(not(test))]
+        assert_eq!(
+            T::Currency::free_balance(&reward_account),
+            0u128.saturated_into(),
+        );
+        #[cfg(test)]
+        assert_eq!(
+            T::Currency::free_balance(&reward_account),
+            reward_before.checked_rem(&o.into()).unwrap(),
+        );
```

Comment:
Tying this to the `test` attribute seems like trouble. I think what we're really trying to express here is this:

```suggestion
        let remainder = reward_before.checked_rem(&o.into()).unwrap();
        let expected = if remainder < T::Currency::minimum_balance() {
            0u8.into()
        } else {
            remainder
        };
        assert_eq!(T::Currency::free_balance(&reward_account), expected);
```

Reply:
I assume the `minimum_balance` is a lot higher than the numbers from `2` to `10`. 

```bash
./target/release/zeitgeist benchmark pallet --chain=dev --steps=10 --repeat=1000 --pallet=zrml_global_disputes --extrinsic=reward_outcome_owner_with_funds --execution=Native --heap-pages=4096 --template=./misc/weight_template.hbs --output=./dev/null

Thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `0`,
 right: `2`', /home/sea212/Documents/github/zeitgeist/zrml/global-disputes/src/benchmarks.rs:286
```

With your suggestion I suspect that we get the same error as above.

Reply:
But I can try it out with `steps > 1`.

Reply:
Without changing to the new polkadot version, but with `steps > 1` it seems to work.